CMakeLists.txt

```diff
@@ -525,38 +567,58 @@ if(BUILD_TOOLS AND WIN32)
 endif()
 
 #--- DDSView sample
+
 if(BUILD_SAMPLE AND BUILD_DX11 AND WIN32)
   list(APPEND TOOL_EXES ddsview)
-
+  Set(DDS_SRC_L
+    DDSView/ddsview.cpp
+    DDSView/ddsview.rc)
+  
+macro(call_mingw_fxc_fx work_dir input_filename output_filename entry_point var_name semantic)
+  set(_fname ${work_dir}/${output_filename})
+  call_mingw_fxc("${work_dir}" "${input_filename}.fx"
+      "${output_filename}" "${entry_point}"
+	  "${var_name}" "${semantic}" "${_fname}")
+  list(APPEND DDS_SRC_L "${_fname}")
+endmacro()
+  
   if(NOT COMPILED_DDSVIEW_SHADERS)
       if(USE_PREBUILT_SHADERS)
           message(FATAL_ERROR "ERROR: Using prebuilt shaders requires the COMPILED_DDSVIEW_SHADERS variable is set")
       endif()
-      set(COMPILED_DDSVIEW_SHADERS ${CMAKE_CURRENT_BINARY_DIR}/Shaders/Compiled)
+      set(COMPILED_DDSVIEW_SHADERS ${CMAKE_CURRENT_BINARY_DIR}/Shadersd/Compiled)
```

Comment:
There's no file name collisions here, so I don't think you need a distinct folder name here.

Reply:
typo.